sos/policies/distros/suse.py

```diff
@@ -62,6 +62,7 @@ class OpenSuSEPolicy(SuSEPolicy):
     distro = "OpenSuSE"
     vendor = "SuSE"
     vendor_urls = [('Community Website', 'https://www.opensuse.org/')]
+    os_release_file = '/etc/SUSE-brand'
```

Comment:
Same here wrt missing `os_release_name(s)`.

Reply:
This is valid, if we're only looking for the location of the file as is, and no `id`/`name`